ClientRequirments.md

```diff
@@ -0,0 +1,37 @@
+# Platform Requirements for Fluid Framework Client Packages
+
+These are the platform requirements for the current version of Fluid Framework Client Packages.
+These are intentionally quite strict as they have to be supported for the entire LTS timeline for LTS versions.
+These restrictions can be loosened over time, adding support for specific configurations desired by users of the client packages.
```

Comment:
"can be loosened" is confusing (and maybe ambiguous). Do we plan to loosen requirements? Or expand supported configurations in ways not explicitly described here. Also, should this document be version specific?

Reply:
Maybe I'm missing something, but don't "loosening the requirements" and "expanding supported configurations" mean exactly the same thing? Both are changes to this document that cause a strict superset of configurations previously supported to be classified as supported by strictly reducing the constrains that are included in this document on what counts as supported.

Reply:
> should this document be version specific?

This document, by its nature of being included in the repo (and likely included in package readmes) applies to what ever version of the packages its included with, and could theoretically be changed between versions to add support for new configurations (along with any needed code and test changes which could be merged together).

It is structured to reduce the need to update this document over time to help prevent it from getting out of date, but it will have to be updated for the next major version (FF 3.0) as it will almost certainly be targeting different versions of everything.